doc/nrf/protocols/matter/getting_started/advanced_kconfigs.rst

```diff
@@ -141,3 +141,11 @@ By default this Kconfig option is set to 1 second.
 .. note::
   The :kconfig:option:`CONFIG_CHIP_FACTORY_RESET_ERASE_NVS` Kconfig option is set to ``y`` by default.
   To disable removing application-specific non-volatile data when the :kconfig:option:`CONFIG_CHIP_LAST_FABRIC_REMOVED_ERASE_AND_REBOOT` Kconfig option is selected, set the :kconfig:option:`CONFIG_CHIP_FACTORY_RESET_ERASE_NVS` Kconfig option to ``n``.
+
+Read Client functionality
+=========================
+
+Read Client functionality needs to be enabled if the device needs to read attributes from another device in the Matter network.
+To enable it set the :kconfig:option:`CONFIG_CHIP_ENABLE_READ_CLIENT` to ``y`` and to ``n`` to disable.
+In Matter samples we have disabled this functionality by default for all samples except :ref:`matter_light_switch_sample`, and :ref:`matter_bridge_app` because both of them need to read attributes from the bound devices.
```

Comment:
It will be hard to track it with time e.g. which examples have or not have this capability.

I would remove 150 and 151 (which is dup of 149).

Maybe just say that this config is disabled by default.